### DISCUSSION TITLE
KodAmbassador & community sesh #1

### DISCUSSION THREAD

**Discussion by damskyftw** [kodadot/nft-gallery#3223] [KodAmbassadors & community stuff]
Hello fellow anons. We are very soon to have first KodAmbassador and community session, which should more of a chilled out time between community managers, founders and community members. Aswell, if you would like to ask any questions regarding KodaDot, feel free to ask either by chat, or raise your hand and you will be given word.

![image](https://user-images.githubusercontent.com/92786519/174603927-a2399835-18cb-4bc8-890c-6f9a0bef838c.png)

As you might’ve seen, we’ve onboarded couple new ambassadors into trial mode. Trial mode means, that after some time, if ambassador is with us for some time and proves his interest about the project + delivers couple stuff that was promised, we will consider hiring that ambassador as official KodaDot ambassador and will be given more responsibility.

_The idea here is for each ambassador to present the work they’ve done, receive feedback from the community and continue with their activity in KodaDot. Currently, ambassadors will continue to be paid by contribution, that is for example article, video, translation, in person event etc._

-------------

### I would kindly ask all the ambassadors to write a bit about themselves and prepare some agenda or their work on the session. We usually go by order regarding who writes their post in this discussion, so feel free to be first!

This way, we can honestly evaluate the value of work provided and openly distribute the rewards.

**Agenda from my side commi{;'ł<*~‘{*&ł‚(sound of pc glitching)(no agenda, its community sesh!)

-------------

### Anyway, my 2 bits are:

- lets talk on previous events
- where are ambassadors from and how it looks with in-person events
- how did you find out about kodadot and why do you stay
- listening to the community, what would you improve, what you really dont like? i’m open ears
- would you like to represent KodaDot at your local events?
- and so on..
- how are nfts doing in the bear market?
- Are we in bear market?
- How’s your haircut? Fresh or halway cut?
- substack

**Comment by AfroCodec** [comment]
look forward creating those TikTok shill vids! 🚀  Koda Up 👐

**Comment by lightsonemusic** [comment]
Love it 👏🥰

**Comment by DeluneClaire** [comment]
Claire's minute:

- Continue on Damsky's talk
- Ambassadors and my thoughts
- TikTok
